### ISSUE TITLE
input_widget.dart:321:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.

### ISSUE THREAD

**Issue by hendrikdewilde** [natintosh/intl_phone_number_input#323]
**Describe the bug**
/src/flutter/.pub-cache/hosted/pub.dartlang.org/intl_phone_number_input-0.7.0+2/lib/src/widgets/input_widget.dart:321:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/src/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {

**Package version**
intl_phone_number_input: ^0.7.0+2

**Flutter version**
Flutter 3.0.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision fb57da5f94 (10 hours ago) • 2022-05-19 15:50:29 -0700
Engine • revision caaafc5604
Tools • Dart 2.17.1 • DevTools 2.12.2

Running flutter doctor...
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.0.1, on Microsoft Windows [Version 10.0.22000.675], locale en-ZA)
[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.2.0)
[√] Android Studio (version 2021.1)
[√] VS Code (version 1.67.1)
[√] Connected device (3 available)
[√] HTTP Host Availability

• No issues found!

**To Reproduce**
1. Pub get
2. Start Project (Run main.dart)

**Expected behavior**
No "Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null" displayed

**Screenshots**

** Targeted platforms (please complete the following information):**
 - OS [Android]
 - Web [❌]

**Additional context**

**Comment by alper-efe-sahin** [comment]
Same on iOS. Package needs update...

**Comment by NguyenPhatBG** [comment]
Same for me.

**Comment by AbdOoSaed** [comment]
```
  intl_phone_number_input:
    git:
      url: https://github.com/AbdOoSaed/intl_phone_number_input
```

**Comment by darkstarx** [comment]
> ```
>   intl_phone_number_input:
>     git:
>       url: https://github.com/AbdOoSaed/intl_phone_number_input
> ```

@AbdOoSaed make the PR please.

**Comment by iamvaidarbhi** [comment]
@hendrikdewilde facing same issue.. In my code , this package having latest version `intl_phone_number_input: ^0.7.0+2` , yet I am also unable to resolve run time error.

@AbdOoSaed please clarify your comment.

**Comment by mwawrusch** [comment]
> 

@iamvaidarbhi  he cloned and fixed it in his github, and that references his source vs this till the library is fixed.

**Comment by abdullah017** [comment]
Is there any update or solution for this problem?

**Comment by SimonH-Zuma** [comment]
Raises the question of how the community at large get pull requests into a public library when the owner seems to have gone AWOL. This library is well used (has 450+ likes pn pub dev). Even if we needed 10 independent code reviews at least we could get some progress here (I'd happily provide one of the code reviews). 

How can Dart/Flutter progress if devs can't trust 'popular' public libraries not to become stale??

**Comment by antoniok90** [comment]
any news about this package?

**Comment by tiancaidao** [comment]
I created this pull request to fix this issue on Flutter 3.0

https://github.com/natintosh/intl_phone_number_input/pull/336

Someone approve it?

**Comment by bynalab** [comment]
> I created this pull request to fix this issue on Flutter 3.0
> 
> #336
> 
> Someone approve it?

Hi @tiancaidao, Your PR has been merged. Thanks.
I'm closing this issue now.
